Lib/test/test_file.py

```diff
@@ -216,6 +216,18 @@ def testSetBufferSize(self):
         with self.assertWarnsRegex(RuntimeWarning, 'line buffering'):
             self._checkBufferSize(1)
 
+    def testDefaultBufferSize(self):
+        f = self.open(TESTFN, 'wb')
+        blksize = f.raw._blksize
+        f.write(bytes([0] * 5_000_000))
```

Comment:
f.write(b"\0" * 5_000_000)

Should be quite a bit faster to do the same thing (Avoids constructing a 5 million element list, then converting each element one at a time from integer to byte and copying across to the `bytes`.